src/config/config.cc

```diff
@@ -201,6 +201,8 @@ Config::Config() {
        new EnumField<rocksdb::CompressionType>(&rocks_db.compression, compression_types,
                                                rocksdb::CompressionType::kNoCompression)},
       {"rocksdb.compression_level", true, new IntField(&rocks_db.compression_level, 32767, INT_MIN, INT_MAX)},
+      {"rocksdb.nocompression_for_first_n_levels", false,
+       new IntField(&rocks_db.nocompression_for_first_n_levels, 2, 0, 7)},
```

Comment:
I'm wondering why it's 7 here? maybe we can add some comments.

Reply:
kvrocks support only 7 levels of leveled compaction, ok i stay a comment

Reply:
fixed

Reply:
I think rather than a comment, we can use a `constexpr` valuable like `KVROCKS_MAX_LSM_LEVEL = 7`, and use it in config and here

Reply:
where should i declare const? in config.h or storage.h?

Reply:
oh i see config.h best place for it

Reply:
Ok to be in config.h

Reply:
fixed